### PR TITLE
fix(pricing,types): remove is_dynamic from model + types

### PR DESCRIPTION
**Pull request by riqwan**
what:

Removing this until we introduce dynamic pricing.

### CODE REVIEW

**Review by shahednasser — APPROVED** [review]
LGTM

File: packages/pricing/src/models/price-rule.ts [hunk 1]
```diff
@@ -39,9 +39,6 @@ export default class PriceRule {
   })
   rule_type: RuleType
 
-  @Property({ columnType: "boolean", default: false })
-  is_dynamic: boolean
-
```
Comment (pKorsholm) [inline packages/pricing/src/models/price-rule.ts:44]: question: should this require an additional migration? or do we want to keep the db column but remove the property here?

**Review by pKorsholm — APPROVED** [review]
lgtm, just a minor change wrt. migrations

File: packages/pricing/src/models/price-rule.ts (hunk 1 above)
Comment (riqwan) [reply packages/pricing/src/models/price-rule.ts:44]: lets keep the migration there for now, we can always remove it at a later point.

**Pull request merged**